application/src/main/java/run/halo/app/core/extension/reconciler/PluginReconciler.java

```diff
@@ -629,7 +629,13 @@ Path toPath(String pathString) {
         if (StringUtils.isBlank(pathString)) {
             return null;
         }
-        return Paths.get(URI.create(pathString).getPath());
+        String separator = File.separator;
+        String processedPathString = pathString.replaceAll("[/\\\\]", separator);
+
+        if (processedPathString.startsWith("file:")) {
+            processedPathString = processedPathString.substring(7);
+        }
+        return Paths.get(processedPathString);
```

Comment:
经过测试，这种方案并不可行。会报错
```
java.lang.IllegalArgumentException: character to be escaped is missing
	at java.base/java.util.regex.Matcher.appendExpandedReplacement(Matcher.java:1019) ~[na:na]
	at java.base/java.util.regex.Matcher.appendReplacement(Matcher.java:997) ~[na:na]
	at java.base/java.util.regex.Matcher.replaceAll(Matcher.java:1181) ~[na:na]
	at java.base/java.lang.String.replaceAll(String.java:2940) ~[na:na]
	at run.halo.app.core.extension.reconciler.PluginReconciler.toPath(PluginReconciler.java:633) ~[main/:na]
	at run.halo.app.core.extension.reconciler.PluginReconciler.buildPluginLocation(PluginReconciler.java:590) ~[main/:na]
	at run.halo.app.core.extension.reconciler.PluginReconciler.determinePluginLocation(PluginReconciler.java:754) ~[main/:na]
	at run.halo.app.core.extension.reconciler.PluginReconciler.lambda$ensurePluginLoaded$19(PluginReconciler.java:620) ~[main/:na]
```

不过如果最终结果是返回 Path 而不需要经过 URI 转换的话，可以直接修改为
```suggestion
       return Paths.get(pathString);
```

测试示例代码不再会报错。

Reply:
出错的 pathString 值是什么

Reply:
在试试

Reply:
> 在试试

ok，没有问题。